testscommon/factory/blockChainHookHandlerFactoryStub.go

```diff
@@ -3,22 +3,19 @@ package factory
 import (
 	"github.com/multiversx/mx-chain-go/process"
 	"github.com/multiversx/mx-chain-go/process/smartContract/hooks"
+	"github.com/multiversx/mx-chain-go/testscommon"
 )
 
-// BlockChainHookHandlerFactoryStub -
-type BlockChainHookHandlerFactoryStub struct {
-	CreateBlockChainHookHandlerCalled func(args hooks.ArgBlockChainHook) (process.BlockChainHookHandler, error)
+// BlockChainHookHandlerFactorySub -
+type BlockChainHookHandlerFactorySub struct {
```

Comment:
Why do we have testscommon/factory/blockChainHookHandlerFactoryMock.go + testscommon/factory/blockChainHookHandlerFactoryStub.go

They could be merged into a single one

Reply:
removed the Stub